projects/1.18/assets/enlightened-end/nourished_end/lang/zh_cn.json

```
@@ -0,0 +1,292 @@
+{
+  "block.nourished_end.verdant_planks": "翠绿木板",
+  "item.nourished_end.stalker_eye": "潜行者之眼",
+  "block.nourished_end.end_iron_ore": "末地铁矿石",
+  "block.nourished_end.orange_shell_tiles": "橙色壳砖",
+  "gamerule.radiationDamage": "辐射伤害",
+  "block.nourished_end.abyssal_shelf_fungus": "毒多孔菌",
+  "block.nourished_end.stripped_verdant_hyphae": "去皮翠绿菌核",
+  "advancements.radiation_fever.descr": "获得足够的放射性来获得致命的辐射伤害",
+  "item.minecraft.potion.effect.illuminating": "发光药水",
+  "block.nourished_end.seldge_patch_air": "塞尔奇空气补丁",
+  "block.nourished_end.verdant_moss": "翠绿苔藓",
+  "block.nourished_end.seldge_slab": "塞尔奇台阶",
+  "advancements.just_so_lucky.descr": "在只有半颗心时吃幸运之根",
+  "item.nourished_end.rye_bread": "末地黑麦面包",
+  "block.nourished_end.end_rubble": "末地碎石",
+  "item.minecraft.lingering_potion.effect.equalized_potion": "滞留型平衡药水",
+  "block.nourished_end.tube_root": "幸运之根",
+  "subtitles.ender_slime_place": "方块放置",

```

原版中的字幕（右下角那个）是有格式的：
 - `主体：行为`，其中`主体`有时略去

如：
 - `美西螈：啾啾`
*美西螈的静息音效*
 - `怪异的噪声`
*洞穴音效*

```suggestion
  "subtitles.ender_slime_place": "方块：被放置",
```